app/Services/User/AuthService.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Services\User;
+
+use Illuminate\Support\Facades\Hash;
+use App\Models\User;
+use Illuminate\Support\Facades\Auth;
+
+/**
+ * Class AuthService.
+ */
+class AuthService
+{
+    public function register(array $data): string
+    {
+        $data['password'] = Hash::make($data['password']);
+        $user = User::create($data);
+        $token = $user->createToken('LaravelClassToken')->accessToken;
```

Comment:
That 'LaravelClassToken', we can move it inside the env file.

We can have auth_token = 'LaravelClassToken'